### PR TITLE
Cover changing content targetting :has-slotted and :has-slotted(...)

### PR DESCRIPTION
**Pull request by Westbrook**
@keithamus these are much more complicated for me to compile in my brain, but I'm pretty confident this covers content swapping for the various cases in question here.

### CODE REVIEW

File: css/css-scoping/has-slotted-changing-001.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      assert_equals(styles.getPropertyValue('color'), green);
+      const div = document.createElement('div');
+      test.append(div);
+      styles = getComputedStyle(target);
+      assert_equals(styles.getPropertyValue('color'), fuchsia);
+      const node = document.createTextNode(' ');
+      test.replaceChildren(node);
+      styles = getComputedStyle(target);
+      assert_equals(styles.getPropertyValue('color'), blue);
+      test.innerHTML = '';
```
Comment (Westbrook) [inline css/css-scoping/has-slotted-changing-001.html:56]: Now with more diverse DOM insertion/update APIs.

File: css/css-scoping/has-slotted-changing-001.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML>
```
Comment (keithamus) [inline css/css-scoping/has-slotted-changing-001.html:1]: Need to rename these files to `.tenatative`.

Comment (Westbrook) [reply css/css-scoping/has-slotted-changing-001.html]: Done and done.

**Comment by Westbrook** [comment]
Is the "wpt-chrome-dev-stability" job telling me I've added tests that fail in Chrome? It talks about "flakes", but they fail regularly, as expected, which leaves me unsure what else is needed to get this green.

Thoughts?

**Review by keithamus — APPROVED** [review]
Looks like a good basis for us to continue with. Let's merge and refine.

**Pull request merged**